### PR TITLE
Only add JavaScript: The Right Way

### PR DESCRIPTION
**Pull request by jamesxv7**
New section created, Browser Compatibility. JavaScript: The Right Way and GitHub Awesome Lists added to respective sections.

### CODE REVIEW

**Comment by NARKOZ** [comment]
Could you leave just `JavaScript: The Right Way` link?

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ coding style guides and development practices across the web.
   + [Favicon](#favicon)
   + [JSON](#json)
   + [Writing](#writing)
-
+
```
Comment (NARKOZ) [inline README.md]: whitespace :scissors:

**Comment by jamesxv7** [comment]
Done!

File: README.md [hunk 2]
```diff
@@ -39,7 +39,6 @@ coding style guides and development practices across the web.
   + [Favicon](#favicon)
   + [JSON](#json)
   + [Writing](#writing)
-
```
Comment (NARKOZ) [inline README.md:42]: line removed

**Comment by jamesxv7** [comment]
I believe now is ready. Sorry for the back-and-forth.

**Comment by NARKOZ** [comment]
Thank you! 907a81c51fb40a879cc9dde8b63ebc602f429345

**Pull request closed without merging**